components/lvgl_esp32_drivers/lvgl_tft/rm67162.c

```diff
@@ -72,9 +72,9 @@ void rm67162_init(void)
 
 	//Reset the display
 	gpio_set_level(RM67162_RST, 0);
-	vTaskDelay(300 / portTICK_RATE_MS);//default 100ms
+	vTaskDelay(300 / portTICK_PERIOD_MS);//default 100ms
 	gpio_set_level(RM67162_RST, 1);
-	vTaskDelay(300 / portTICK_RATE_MS);//default 100ms
```

Comment:
dont change file or function from another auther in your PR.

Reply:
because my file did'nt compiled  , I should be change it .